Test failures due to incorrect guess on needing "userxattr"

I'm trying out the Flake instructions. I get this error when building `nix-snapshotter`:

```
--- FAIL: TestSnapshotter (0.01s)
    --- FAIL: TestSnapshotter/no_opt (0.01s)
        --- FAIL: TestSnapshotter/no_opt/TestSnapshotterView (0.00s)
            snapshotter_overlay_test.go:389:   []string{
                        "lowerdir=/build/TestSnapshotterno_optTestSnapshotterView20029197"...,
                +       "userxattr",
                        "volatile",
                  }
        --- FAIL: TestSnapshotter/no_opt/TestSnapshotterOverlayMount (0.00s)
            snapshotter_overlay_test.go:389:   []string{
                        "lowerdir=/build/TestSnapshotterno_optTestSnapshotterOverlayMount"...,
                        "upperdir=/build/TestSnapshotterno_optTestSnapshotterOverlayMount"...,
                +       "userxattr",
                        "workdir=/build/TestSnapshotterno_optTestSnapshotterOverlayMount2"...,
                  }
    --- FAIL: TestSnapshotter/AsynchronousRemove (0.00s)
        --- FAIL: TestSnapshotter/AsynchronousRemove/TestSnapshotterView (0.00s)
            snapshotter_overlay_test.go:389:   []string{
                        "lowerdir=/build/TestSnapshotterAsynchronousRemoveTestSnapshotter"...,
                +       "userxattr",
                        "volatile",
                  }
        --- FAIL: TestSnapshotter/AsynchronousRemove/TestSnapshotterOverlayMount (0.00s)
            snapshotter_overlay_test.go:389:   []string{
                        "lowerdir=/build/TestSnapshotterAsynchronousRemoveTestSnapshotter"...,
                        "upperdir=/build/TestSnapshotterAsynchronousRemoveTestSnapshotter"...,
                +       "userxattr",
                        "workdir=/build/TestSnapshotterAsynchronousRemoveTestSnapshotterO"...,
                  }
```

I'm not sure of the root cause of the error; the machine is running k3s on NixOS, with kernel 5.15.114.

--- a/package.nix
+++ b/package.nix
@@ -18,7 +18,7 @@ let
         ".tar"
       ];
     };
-    vendorSha256 = "sha256-fTF902RO5r6mui+7ZQWASM4xUWema4gNcShmdA3JVBQ=";
+    vendorSha256 = "sha256-gK7FAK6RQ9hL3K4PY2f2z4nBDBJXRctUZbVHuVqOIYE=";
     passthru = { inherit buildImage; };
   };
 

--- a/pkg/nix/snapshotter_overlay_test.go
+++ b/pkg/nix/snapshotter_overlay_test.go
@@ -18,9 +18,9 @@ import (
 	"github.com/containerd/containerd/pkg/testutil"
 	"github.com/containerd/containerd/snapshots"
 	"github.com/containerd/containerd/snapshots/overlay"
+	"github.com/containerd/containerd/snapshots/overlay/overlayutils"
 	"github.com/containerd/containerd/snapshots/storage"
 	"github.com/containerd/containerd/snapshots/testsuite"
-	"github.com/docker/docker/daemon/graphdriver/overlayutils"
 	"github.com/google/go-cmp/cmp"
 	"github.com/stretchr/testify/require"
 )

